wiki/Beatmapping/Overmapping/id.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Overmapping
+
+**Overmapping** adalah istilah yang digunakan ketika [objek ketukan](/wiki/Hit_object) dipetakan sedemikian rupa sehingga membuat [tingkat kesulitan](/wiki/Beatmap/Difficulty) berlawanan dengan lagu.
```

Comment:
```suggestion
**Overmapping** adalah istilah yang digunakan ketika [objek ketukan](/wiki/Hit_object) di-map sedemikian rupa sehingga membuat [tingkat kesulitan](/wiki/Beatmap/Difficulty) berlawanan dengan lagu.
```
`di-map` is better term since `mapped` isn't literally mapped